components/dashboard/dashboard.go

```diff
@@ -240,7 +250,7 @@ func (d *Dashboard) deployConsoleLink(cli client.Client, owner metav1.Object, pl
 	domainIndex := strings.Index(consoleRoute.Spec.Host, ".")
 	consoleLinkDomain := consoleRoute.Spec.Host[domainIndex+1:]
 	err := common.ReplaceStringsInFile(pathConsoleLink, map[string]string{
-		"<dashboard-url>": "https://rhods-dashboard-" + namespace + "." + consoleLinkDomain,
+		"<dashboard-url>": "https://" + componentName + "-" + namespace + "." + consoleLinkDomain,
```

Comment:
I've just tested this in my cluster, I lack the whole context so I'm not sure where componentName comes from, but in ODH, the formatted url is: `https://dashboard-opendatahub.apps...` when the actual url is ``https://odh-dashboard-opendatahub.apps...`, it's missing the odh prefix here.

Reply:
need to know how you create this URL , is it hardcoded by dashboard to always set to https://odh-dashboard-opendatahub.* for ODH, or if that might change in different cases

Reply:
updated the code to hardcode ODH for `odh-dashboard`